bindings/matrix-sdk-crypto-js/package.json

```
@@ -38,6 +38,8 @@
     "scripts": {
         "build": "cross-env RUSTFLAGS='-C opt-level=z' wasm-pack build --release --target nodejs --scope matrix-org --out-dir ./pkg",
         "test": "jest --verbose",
-        "doc": "typedoc --tsconfig ."
+        "doc": "typedoc --tsconfig .",
+        "prepublish": "$npm_execpath run build && $npm_execpath run test",
```

```suggestion
        "prepublish": "$npm_execpath run build && $npm_execpath run test && wasm-pack pack",
```

(not sure this might need to be in the subdirectory?)

---

`wasm-pack publish` will _pack_ and _publish_, so it's not necessary to run `wasm-pack pack`.

Quoting [from the officlal `wasm-pack` documentation](https://rustwasm.github.io/docs/wasm-pack/commands/pack-and-publish.html):

> The `pack` command creates a tarball from the `pkg` directory and the `publish` command creates a tarball from the `pkg` directory **and** publishes it to the NPM registry.

---

yeah, the only point to run `pack` was to build the tarball for the upload ... unfortunately publish does that internally but doesn't give you a path to the tarball after ...

---

Will fix that.

---

Done.